errors.js

```diff
@@ -25,6 +25,12 @@ var DatacenterRequired = TypedError({
         'SUGGESTED FIX: update the `fetchConfig()` callsite.\n'
 });
 
+var NonexistantKeyPath = TypedError({
+    type: 'nonexistant.key.path',
+    message: 'attempting to get a nonexistant keyPath.\n' +
```

Comment:
message: 'attempted to get a key in ZeroConfig that does not exist.\n' +
   'Expected the key {keyPath} to have a defined value.\n' +
   'SUGGESTED FIX: Update your configuration files to define the key.\n'